db/migrate/20181217150935_change_requires_action_to_be_datetime.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeRequiresActionToBeDatetime < ActiveRecord::Migration[5.2]
+  def change
+    add_column :solution_mentorships, :requires_action_since, :datetime, null: true
+
+    SolutionMentorship.includes(:solution).where(requires_action: true).each do |sm|
+      d = sm.solution.last_updated_by_user_at
+      d = sm.updated_at if sm.updated_at > d
+      sm.update(requires_action_since: d)
+    end
+
+    #remove_column :solution_mentorships, :requires_action
```

Comment:
What is this commented out line for? Can we delete this?

Reply:
I don't want to lose the column in case the migration logic turns out to be wrong and then we have no way of getting back to the real data.